docs/customization.md

```diff
@@ -281,3 +281,21 @@ ReactDOM.render(
 
 [api-reference-editor-state]: https://facebook.github.io/draft-js/docs/api-reference-editor-state.html
 [custom actions]: https://github.com/globocom/megadraft/blob/master/src/actions/default.js
+
+
+### Modal
+
+There's a modal option in the sidebar that can show plugins inside. By default Megadraft doesn't show the modal. To turn on this option just pass the desirable number (as example above) of plugins you want to show in sidebar to the prop `sideBarMaxNumberPlugins`. If it's preferable this option can be not used. Just need to pass `null` in the option `sideBarMaxNumberPlugins` of `renderSidebar`.
```

Comment:
Please include a brief description on the "Editor props" section.

And also, how about the text below?

```
Handling too many plugins

By default, plugin buttons are shown on a vertical sidebar. This may be a bit
cumbersome when there is a lot of enabled plugins.

To overcome this, the prop `maxSidebarButtons` limits the number of buttons
displayed on the sidebar. When the limit is reached an extra button will appear
and when clicked it will open a modal window with the full button list.

Passing a number to the `maxSidebarButtons` prop will limit the number of
buttons displayed on the sidebar.

Omitting this prop or passing a `null` value forces all buttons to be displayed
on the sidebar.
```